rfc-009-transparent-code-bases.md

```diff
@@ -0,0 +1,20 @@
+# Transparent Codebases
+
+## Summary
+
+Teams are expected to publish details on accessing their source code.
+
+## Problem
+
+It is currently difficult for teams to see what other teams are working on, and gain insights, provide feedback and generally for us as a company to learn from each other.
+
+Even in the case where projects are hosted on madetech github, it's hard to filter through the large number of repositories.
+
+## Proposal
+
+Teams must publish, to a centralised location, links to details on how any Made Tech engineer can access their customer code.
```

Comment:
Where would we define this centralised location, the handbook, here, elsewhere?

Reply:
Potentially a sensitive list so perhaps a link to a private spreadsheet (shared with Made Tech org) from the handbook?